docs/source/onboarding_resources/computing/index.rst

```diff
@@ -0,0 +1,359 @@
+
+.. _cluster_access_files:
+
+File Systems and Storage 
+------------------------
+
+The cluster can be confusing with where to store code and data. Our 
+team has created some best practices to use for data storage. 
+
+For code, please create a new directory under :code:`/ihme/code` with your 
+username. For example, this might be :code:`/ihme/code/lutzes`. 
+You should clone GitHub respositories to this location and have 
+all Jupyter notebooks and other code stored here. 
+
+For data files, there are two locations based on the size of the 
+data file. 
+
+#. For small data files, store these on GitHub in the same location as your code. A "small" file is something without thousands of rows of data. Examples might include: a list of nicknames, disease severity proportions by age/sex group, or drug efficacy data. 
+#. For large files, store these in a shared location on the cluster. Considering making a new folder for each project for data storage. 
+#. When you deciding where to store data, please also consider any data restrictions that might exist. 
+
+Regardless of where you store data, it is important to track updates 
+to data files carefully. Engineers might copy and paste a file into a 
+new location, so updating the file might not actually change what is 
+being used in the sim. Therefore, follow these steps: 
+
+#. Use the naming conventions below to ensure consistency. 
+#. Always version up rather than replacing a data file (e.g., create a new file with the current date rather than just replacing with a different file of the same name). 
+#. Include the exact file name and location in the docs. This means if you version up a data file, you will need to update the docs to reflect the new name. This ensures the engineers are aware of any changes. 
+
+For consistency, please use this naming convention for all files: :code:`FILENAME_20230309.ext`. 
+For example, this might be :code:`heart_failure_proportions_20230310.csv` 
+
+.. _cluster_access_bash:
+
+Your Bash Files 
+---------------
+
+Bash files contain commands you would enter into a command line, but 
+specifically ones you will use very often. For example, every time you 
+want to open a Juptyer session, the cluster needs certain information 
+and requirements. To find this information, it looks in your Bash files 
+rather than asking you to enter the same information every time. 
+
+However, Bash files can be confusing since it is less obvious when the 
+information is being used or what it is used for. Therefore, we have 
+provided a copy-and-paste formatting for information to be added to 
+your Bash files. If you would like more information and what this 
+code means and why we need it, this `Hub page <https://hub.ihme.washington.edu/pages/viewpage.action?pageId=83205636#UsingtheCluster&Bash-Bash>`_ has a number of helpful links. 
```

Comment:
Not sure this Hub page adds much. It lists the files under "What you should know" but doesn't explain them.